rspace/src/main/scala/coop/rchain/rspace/history/instances/RadixHistory.scala

```diff
@@ -14,7 +14,7 @@ import scodec.bits.ByteVector
   * History implementation with radix tree
   */
 object RadixHistory {
-  val emptyRootHash: Blake2b256Hash = Blake2b256Hash.fromByteVector(hashNode(emptyNode)._1)
+  val emptyRootHash: Blake2b256Hash = RadixTree.emptyRootHash
```

Comment:
It is just duplicates the same value from `RadixTree`. I think we can eliminate it.